sdcm/utils/remote_logger.py

```diff
@@ -299,6 +300,38 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"
 
 
+class WrongSchedulingLogger(CommandClusterLoggerBase):
+    restart_delay = 30
+
+    @property
+    def _logger_cmd(self) -> str:
+        if not self._cluster.allowed_labels_on_scylla_node:
+            return ''
+
+        wrong_scheduled_pods_on_scylla_node = []
+        scylla_pool = self._cluster.pools[self._cluster.SCYLLA_POOL_NAME]
+
+        nodes = scylla_pool.nodes
+
+        for node in nodes.items:
```

Comment:
These lines could be simplified to the following ones:

```
for node in self._cluster.pools[self._cluster.SCYLLA_POOL_NAME].nodes.items:
    ...
```

Reply:
done